examples/next/pages/ui/hooks/actions/NavigateActions.tsx

```diff
@@ -20,15 +20,9 @@ export const NavigateActions = () => {
   };
   return (
     <View>
-      <Button testId="locationChange" onClick={() => goToAmazon()}>
-        Go to amazon.com
-      </Button>
-      <Button testId="reload" onClick={() => reload()}>
-        Reload
-      </Button>
-      <Button testId="hash" onClick={() => handleHashClick()}>
-        Go to #notes
-      </Button>
+      <Button onClick={() => goToAmazon()}>Go to amazon.com</Button>
+      <Button onClick={() => reload()}>Reload</Button>
+      <Button onClick={() => handleHashClick()}>Go to #notes</Button>
```

Comment:
Corresponding tests have been updated to use accessible role queries instead of test IDs.